### PR TITLE
Sink some arrays

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

**Comment by janisozaur** [comment]
Build fails because of https://github.com/discordapp/discord-rpc/commit/e8091f5137495a3b7691878a3be12b38540f1cfb

**Comment by janisozaur** [comment]
Created https://github.com/Microsoft/vcpkg/pull/2904

**Comment by Gymnasiast** [comment]
To me, this seems to lower readability. What is the advantage?

File: src/openrct2/paint/VirtualFloor.cpp [hunk 1]
```diff
@@ -139,6 +131,14 @@ void virtual_floor_paint(paint_session * session)
     //  if our parameters differ we set an edge towards that tile
     for (uint8 i = 0; i < 4; i++)
     {
+        static constexpr const CoordsXY offsets[4] =
```
Comment (ZehMatt) [inline src/openrct2/paint/VirtualFloor.cpp]: Having this in a loop now seems really strange to me.

**Comment by janisozaur** [comment]
@Gymnasiast those are only used in single context, so I argue it makes sense to keep them closer. The other benefit it has it removes two symbols with the same name from global namespace which lets unity builds proceed further.

File: src/openrct2/paint/VirtualFloor.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2/paint/VirtualFloor.cpp]: Yes, can you at least move them out of the loop? Also for legibility reasons.

**Comment by janisozaur** [comment]
This is updated now.

**Pull request merged**